docs/site-example/site.conf

```diff
@@ -164,7 +164,8 @@
   },
 
   autoupdater = {
-    -- Default branch. Don't forget to set GLUON_BRANCH when building!
+    -- Default branch, can be overridden by setting GLUON_AUTOUPDATER_BRANCH when building.
+    -- Set GLUON_AUTOUPDATER_ENABLED to enable the autoupdater by default.
     branch = 'stable',
```

Comment:
add "optional" here, (defaults to the alphabetical first branch)

Reply:
I've made a few more changes to the docs.